frontends/web/src/components/aopp/aopp.tsx

```diff
@@ -38,164 +37,114 @@ const Banner = ({ children }: TProps) => (
-            </Message>
-          </ViewContent>
-          <ViewButtons>
-            <Button danger onClick={aoppAPI.cancel}>{t('button.dismiss')}</Button>
-          </ViewButtons>
-        </View>
-      );
-    case 'inactive':
-      // Inactive, waiting for action.
-      return null;
-    case 'user-approval':
-      return (
-        <View
-          fullscreen
-          textCenter
-          verticallyCentered
-          width="580px">
-          <ViewHeader title={t('aopp.title')} withAppLogo />
-          <ViewContent>
-            <Vasp prominent
-              hostname={domain(aopp.callback)}
-              fallback={(
-                <SimpleMarkup tagName="p" markup={t('aopp.addressRequest', {
-                  host: `<strong>${domain(aopp.callback)}</strong>`
-                })}/>
-              )}
-              withLogoText={t('aopp.addressRequestWithLogo')} />
-          </ViewContent>
-          <ViewButtons>
-            <Button primary onClick={aoppAPI.approve}>{t('button.continue')}</Button>
-            <Button secondary onClick={aoppAPI.cancel}>{t('dialog.cancel')}</Button>
-          </ViewButtons>
-        </View>
-      );
-    case 'awaiting-keystore':
-      return (
-        <Banner>{t('aopp.banner')}</Banner>
-      );
-    case 'choosing-account': {
-      const options = aopp.accounts.map(account => {
-        return {
-          text: account.name,
-          value: account.code,
-        };
-      });
-      return (
-        <form onSubmit={this.chooseAccount}>
-          <View
-            fullscreen
-            textCenter
-            verticallyCentered
-            width="580px">
-            <ViewHeader title={t('aopp.title')}>
-              <Vasp hostname={domain(aopp.callback)} />
-            </ViewHeader>
-            <ViewContent>
-              <Select
-                label={t('buy.info.selectLabel')}
-                options={options}
-                value={accountCode}
-                onChange={e => this.setState({ accountCode: (e.target as HTMLSelectElement)?.value })}
-                id="account" />
-            </ViewContent>
-            <ViewButtons>
-              <Button primary type="submit">{t('button.next')}</Button>
-              <Button secondary onClick={aoppAPI.cancel}>{t('dialog.cancel')}</Button>
-            </ViewButtons>
-          </View>
-        </form>
-      );
-    }
-    case 'syncing':
-      return (
+  if (!aopp) {
+    return null;
+  }
+  switch (aopp.state) {
+  case 'error':
+    return (
+      <View
+        fullscreen
+        textCenter
+        verticallyCentered
+        width="580px">
+        <ViewHeader title={t('aopp.errorTitle')}>
+          <p>{domain(aopp.callback)}</p>
+        </ViewHeader>
+        <ViewContent>
+          <Message type="error">
+            <Cancel className={styles.smallIcon} />
+            {t(`error.${aopp.errorCode}`, { host: domain(aopp.callback) })}
+          </Message>
```

Comment:
I removed the `Cancel` and now it looks like this:

![image](https://github.com/user-attachments/assets/ea88f8de-c77d-4d8c-a128-caaa20845158)

Did you mean it is totally unused in the frontend or totally unused in the component ? I'll check if it can be removed globally and do that in a separate commit then.